cmd/sippy/automatejira.go

```diff
@@ -0,0 +1,157 @@
+package main
+
+import (
+	"context"
+	"flag"
+	"fmt"
+	"strconv"
+	"strings"
+	"time"
+
+	"github.com/openshift/sippy/pkg/api"
+	"github.com/openshift/sippy/pkg/api/componentreadiness"
+	crtype "github.com/openshift/sippy/pkg/apis/api/componentreport"
+	"github.com/openshift/sippy/pkg/apis/cache"
+	bqcachedclient "github.com/openshift/sippy/pkg/bigquery"
+	"github.com/openshift/sippy/pkg/componentreadiness/jiraautomator"
+	"github.com/openshift/sippy/pkg/flags"
+	"github.com/openshift/sippy/pkg/util/sets"
+	"github.com/pkg/errors"
+	log "github.com/sirupsen/logrus"
+	"github.com/spf13/cobra"
+	"github.com/spf13/pflag"
+	prowflagutil "sigs.k8s.io/prow/pkg/flagutil"
+)
+
+type AutomateJiraFlags struct {
+	BigQueryFlags                                *flags.BigQueryFlags
+	GoogleCloudFlags                             *flags.GoogleCloudFlags
+	CacheFlags                                   *flags.CacheFlags
+	ComponentReadinessFlags                      *flags.ComponentReadinessFlags
+	JiraOptions                                  prowflagutil.JiraOptions
+	SippyURL                                     string
+	ComponentWhiteListStr                        string
+	ComponentWhiteList                           sets.String
```

Comment:
Industry is moving away from white/black list terminology. We should use allowlist / blocklist

e.g. https://developers.google.com/style/inclusive-documentation#replace

Reply:
ComponentAllowList doesn't seem to convey the same as ComponentWhiteList. I think I am going to use includeComponents.